angular/src/app/containers/map/map.component.ts

```diff
@@ -8,7 +8,7 @@ import * as mapActions from '../actions/map';
 @Component({
   selector: 'dp-map',
   changeDetection: ChangeDetectionStrategy.OnPush,
-  template: `<div dp-leaflet>This will show a map</div>`
+  template: `<div dp-leaflet class="c-map"></div>`
```

Comment:
Als je van `dp-leaflet` een component maakt is dit ook weer iets cleaner; `<dp-leaflet></dp-leaflet>`.